.github/workflows/scripts/ti_build/vulkan.py

```diff
@@ -10,30 +10,34 @@
 from .misc import banner, get_cache_home, path_prepend
 from .python import path_prepend
 
-VULKAN_VERSION = "1.4.304.1"
```

Comment:
Let's continue to use a version string please, rather than hard-coding in each download.